MicrowaveOven.playground/Contents.swift

```diff
@@ -1 +1,67 @@
+import Foundation
+import PlaygroundSupport
 
+PlaygroundPage.current.needsIndefiniteExecution = true
+
+class MicrowaveOven {
+    
+    enum Power {
+        case w100
+        case w200
+        case w500
+    }
+    
+    var maxTimeW100: Int = 1200
+    var maxTimeW200: Int = 900
+    var maxTimeW500: Int = 600
+    
+    var timer: Timer?
+    var setTime: Int = 0
+
+    func start(power: Power, setTime: Int) -> Bool {
+        self.setTime = setTime
+        switch power {
+        case .w100:
+            if self.setTime > maxTimeW100{
+                print("温め時間は\(maxTimeW100)秒以下でお願いします。")
+                return false
+            }
+        case .w200:
+            if self.setTime > maxTimeW200{
+                print("温め時間は\(maxTimeW100)秒以下でお願いします。")
```

Comment:
```suggestion
                print("温め時間は\(maxTimeW200)秒以下でお願いします。")
```
こちら、仕様的に↑が正しいではないでしょうか?